rustls/tests/unbuffered.rs

```diff
@@ -888,3 +888,41 @@ fn make_connection_pair(
     let server = UnbufferedServerConnection::new(Arc::new(server_config)).unwrap();
     (client, server)
 }
+
+#[test]
+fn server_receives_handshake_byte_by_byte() {
```

Comment:
This commit has the message: "See comment about how NeedMoreTlsData's num_bytes field is currently
unused.", but I don't see a comment about this in the diff. Am I missing something or should the commit message be adjusted?

Reply:
cc @ctz

Reply:
Please could you remove that bit from the commit message?

Reply:
done